src/routes/+page.svelte

```diff
@@ -3,20 +3,37 @@
 		ArrowDown,
 		ChevronRight,
 		CodeBracket,
-		DevicePhoneMobile
+		DevicePhoneMobile,
+		Cloud,
+		Swatch
 	} from "@inqling/svelte-icons/heroicon-24-solid";
 	import { Sparkles, Window } from "@inqling/svelte-icons/heroicon-24-outline";
 	import Button from "$ui/Button.svelte";
 	import Section from "$components/Section.svelte";
 	import MagneticElement from "$components/MagneticElement.svelte";
 	import Mouse3DTilting from "$components/Mouse3DTilting.svelte";
+	import { scrollTo } from "$ts/scroll";
 
-	function scrollTo(selector: string) {
-		const element = document.querySelector(selector);
-		if (element) {
-			element.scrollIntoView({ behavior: "smooth" });
+	const processSections = [
+		{
+			title: "Design",
+			icon: Swatch,
+			description:
+				"We design the graphical interface of your project in accordance with your visual identity, to ensure an optimal user experience. "
+		},
+		{
+			title: "Development",
+			icon: CodeBracket,
+			description:
+				"Our company specializes in transforming your concepts into functional code, utilizing the latest and most advanced technologies in the market."
+		},
+		{
+			title: "Hosting",
+			icon: Cloud,
+			description:
+				"We take care of all the steps hosting options for your project so that you can focus on your business rather than the technical tasks associated with hosting."
```

Comment:
Maybe precise that we're open for both at-Vercel hosting and on-premise (on the client's servers)?

Reply:
Added, are you ok with that?